ajax/libs/jquery.flipster/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "jquery.flipster",
+  "homepage": "http://brokensquare.com/Code/jquery-flipster/demo/",
+  "description": "Responsive coverfpsterlow-style carousel plugin for jQuery",
+  "filename": "jquery.flipster.min.js",
+  "author": "Adrien Delessert <a@drien.com>",
+  "keywords": [
+    "coverflow",
+    "jquery",
+    "carousel",
+    "flipster",
+    "jquery-plugin",
+    "ecosystem:jquery"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/drien/jquery-flipster.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/drien/jquery-flipster.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
```

Comment:
Do we really need any source files here? It seems dist has the js + css you'd need to use this?

Reply:
As I remember this one and also below one are for old version fetch. 
Here [explanation](https://github.com/cdnjs/cdnjs/pull/12054#pullrequestreview-143372757) for those regexes

Reply:
Hm. IMO, we shouldn't worry about ensuring we can pull in every previous version. if we provide the glob that works for the current release, the auto-update bot will work backwards as far as it can finding matches. I think writing additional globs to pull in files for much older versions complicates things beyond what is required?